SPEC.md

```diff
@@ -2488,7 +2502,7 @@ Example 2:
   }
 
   output {
-    File outputFile = ${output_file_name}
+    File outputFile = output_file_name
```

Comment:
Wait, is this a new syntax for outputs? And does this mean "you can write it either way" or "you have to write it the new way"?

Apologies if this was specified in release notes that I failed to read.